client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -2,428 +2,340 @@
-                                                      data-ng-if="hosting.hostingIpv6 == null"></span>
-                                            </div>
-                                        </div>
-                                    </div>
-                                </li>
-                                <li class="oui-tile__item">
-                                    <div class="oui-tile__definition">
-                                        <div class="d-flex w-100">
-                                            <div>
-                                                <strong class="oui-tile__term d-block"
-                                                        data-translate="hosting_dashboard_service_offer"></strong>
-                                                <span class="text-capitalized mr-2"
-                                                      data-ng-bind="'hosting_dashboard_service_offer_'+ hosting.offer | translate"></span>
-                                                <span class="fa fa-exclamation-triangle text-warning" aria-hidden="true"
-                                                      data-ng-if="hostingProxy.recommendedOffer"
-                                                      data-uib-tooltip="{{:: 'hosting_dashboard_service_offer_recommanded' | translate: { t0: hostingProxy.recommendedOffer } }}"></span>
-                                            </div>
-                                        </div>
-                                    </div>
-                                    <oui-action-menu data-compact data-align="end"
-                                                     data-ng-if="!hosting.isCloudWeb">
-                                        <button class="oui-dropdown-option ng-binding ng-scope" type="button"
-                                                data-ng-click="setAction('offer/upgrade/hosting-offer-upgrade')"
-                                                data-track-on="click"
-                                                data-track-name="web::hostname::general-informations::change-offer"
-                                                data-track-click="action">
-                                                <span data-translate="hosting_dashboard_service_change_offer"></span>
-                                        </button>
-                                    </oui-action-menu>
-                                </li>
-                                <li class="oui-tile__item">
-                                    <div data-ng-if="!hosting.isExpired"
-                                         data-ng-show="hosting.quotaPercent != null && hosting.quotaPercent >= 0">
-
-                                        <strong class="oui-tile__term" data-ng-bind-html="'hosting_dashboard_disk_usage' | translate"></strong>
+        <div class="alert alert-dismissible alert-info" role="alert"
```

Comment:
Could be turned into a [`<oui-message>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/message)?